src/View/Helper/RnaHelper.php

```diff
@@ -83,6 +83,31 @@ protected function loadEntrypoints(?string $plugin = null): ?array
         return $this->cache[$cacheKey] = json_decode(file_get_contents($path), true);
     }
 
+    /**
+     * Patch current request with an empty webroot attribute.
+     * RNA entrypoints already includes the full webroot path.
+     * We are removing the `webroot` attribute in order to prevent additional prefix
+     * when using Html::script and Html::css methods.
+     *
+     * @param callable $callback The callback to exec with patched request.
+     */
+    protected function patchViewRequest(callable $callback)
+    {
+        $view = $this->getView();
+        $request = $view->getRequest();
+        $patched = $request->withAttribute('webroot', '/');
+        $view->setRequest($patched);
+        $res = null;
+
+        try {
+            $res = $callback();
+        } finally {
+            $view->setRequest($request);
+        }
+
+        return $res;
```

Comment:
You might also `return` from within the `try` block, and the `finally` would still get executed _after_ the callback has been executed, but _before_ the returned value is passed to the invoker context. 🤓 

```suggestion

        try {
            return $callback();
        } finally {
            $view->setRequest($request);
        }
```